tests/specs/v2/rumXhrStatusTracking.js

```diff
@@ -102,17 +102,17 @@ describe("RUM status code tracking", function() {
     
         var supportsFetch = browser.execute(function() {
           return window.supportsFetch;
-        }).value;
+        });
     
         if (!supportsFetch) {
           return;
         }
 
-        browser.pause(34000);
+        browser.pause(35000);
```

Comment:
Curious as to why these values needed increasing?

Reply:
When testing locally I was hitting the `test did not wait long enough for ajax requests to be sent to Raygun` so updated these to match the other tests